sec_edgar_downloader/_utils.py

```diff
@@ -72,7 +76,12 @@ def get_filing_urls_to_download(
         )
         edgar_search_url = f"{SEC_EDGAR_BASE_URL}{qs}"
 
-        resp = requests.get(edgar_search_url)
+        with RateLimiter('sec-state') as limiter:
+            while limiter.ready() is False:
+                time.sleep(.1)
```

Comment:
We probably need to use this rate limiter in `download_filings` as well since this function is only focused on obtaining the URLs to download.

https://github.com/jadchaar/sec-edgar-downloader/blob/04310606f262cbff37aba6a677ce06dd4e6caa2c/sec_edgar_downloader/_utils.py#L134